Gemfile.lock

```diff
@@ -19,6 +19,8 @@ GEM
       safe_yaml (~> 1.0)
     jekyll-feed (0.8.0)
       jekyll (~> 3.3)
+    jekyll-last-modified-at (0.3.4)
```

Comment:
Gemfile.lock is recorded when you 'bundle update' and is a reflection of the things that the dependency graph wanted now. The idea in having this in version control is to have what devs test with. We could .gitignore it and have it built every time, but either way, it's probably best to not care too much about this file.